config/config.sample.pythtest.toml

```diff
@@ -3,12 +3,10 @@ listen_address = "127.0.0.1:8910"
 
 [primary_network]
 
-# HTTP(S) endpoint of the RPC node. Public RPC endpoints are usually rate-limited, so a private
-# endpoint should be used in most cases.
+# HTTP(S) endpoint of the RPC node.
 rpc_url = "https://api.pythtest.pyth.network"
 
-# WS(S) endpoint of the RRC node. This is used to subscribe to account changes on the network.
-# This can be omitted when oracle.subscriber_enabled is set to false.
```

Comment:
This was a nice comment, not sure if you removed it because it doesn't hold anymore or not